From c9985217378cba90d83101d12bc462637e730ad8 Mon Sep 17 00:00:00 2001
From: Rich Megginson <rmeggins@redhat.com>
Date: Wed, 31 Jul 2013 10:52:21 -0600
Subject: [PATCH 169/225] fix coverity 11915 - dead code - introduced with fix
 for ticket 346

just get rid of unused variable was_present_null
reviewed by: nhosoi (Thanks!)
(cherry picked from commit ba70aac1991a04455ba83362e22c33f7f6ab227a)
(cherry picked from commit fcf4154b73681752a5ac32954a1d25771a75478e)
(cherry picked from commit 8a6b75fefccb9daf238d26fcc6f320d4505f8251)
---
 ldap/servers/slapd/attr.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/ldap/servers/slapd/attr.c b/ldap/servers/slapd/attr.c
index 9894393..983e286 100644
--- a/ldap/servers/slapd/attr.c
+++ b/ldap/servers/slapd/attr.c
@@ -811,7 +811,6 @@ attr_add_valuearray(Slapi_Attr *a, Slapi_Value **vals, const char *dn)
     int i = 0;
     int numofvals = 0;
     int duplicate_index = -1;
-    int was_present_null = 0;
     int rc = LDAP_SUCCESS;
 
     if (valuearray_isempty(vals)) {
@@ -868,7 +867,7 @@ attr_add_valuearray(Slapi_Attr *a, Slapi_Value **vals, const char *dn)
                 duplicate_string,
                 a->a_type,
                 dn ? dn : "<null>", 
-                (was_present_null ? "duplicate new value" : "value exists"));
+                "value exists");
     }
     return( rc );
 }
-- 
1.8.1.4

